sys/kern/kern_sig.c

```diff
@@ -1239,21 +1199,20 @@ user_sigwait(struct thread *td, const sigset_t * __capability uset,
 	return (0);
 }
 
-int
-copyout_siginfo_native(const _siginfo_t *si, void * __capability info)
+static int
+copyout_siginfo(const siginfo_t *si, void * __capability info)
 {
-	struct siginfo_native si_n;
 
-	siginfo_to_siginfo_native(si, &si_n);
-	return (copyout_c(&si_n, info, sizeof(si_n)));
+	/* XXX: copyoutcap for si_addr and si_sigval.sival_ptr? */
```

Comment:
This is used for things like sigtimedwait(), etc. and I think probably should not be stripping tags.  If there are cases where this is used for cross-process we should probably strip tags in those callers explicitly.

Reply:
Fixed